logprep/processor/geoip_enricher/processor.py

```diff
@@ -70,13 +70,14 @@ def _try_getting_geoip_data(self, ip_string):
         try:
             ip_addr = str(ip_address(ip_string))
             ip_data = self._city_db.city(ip_addr)
+
+            if ip_string is None:
+                return {}
```

Comment:
I think it is too late here... Consider to do this check in the `apply_rules` method

Reply:
And I think if this string is None, we should raise a `ProcessingWarning` because the user thought there should be a value, but there is no value or the filter is not right. However, the user should know this.